src/components/SideDrawer/FieldWrapper.tsx

```diff
@@ -47,9 +51,15 @@ export default function FieldWrapper({
   hidden,
   index,
 }: IFieldWrapperProps) {
-  const [projectId] = useAtom(projectIdAtom, projectScope);
-  const [altPress] = useAtom(altPressAtom, projectScope);
-
+  const [projectId] = useAtom(projectIdAtom, globalScope);
+  const [altPress] = useAtom(altPressAtom, globalScope);
```

Comment:
Please check that the app runs before submitting a PR. The develop branch renames `globalScope` to `projectScope`.

Reply:
Fixed